Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml

```
@@ -11,3 +11,32 @@
   - type: Sprite
     sprite: DeltaV/Objects/Misc/stamps.rsi
     state: stamp-psychologist
+    
+- type: entity
+  name: notary stamp
+  parent: RubberStampBase
+  id: RubberStampNotary
+  description: An old-fashioned seal for marking important documents, made of polished bronze.
+  components:
+  - type: Stamp
+    stampedName: stamp-component-stamped-name-notary
+    stampedColor: "#a81f3d"
+    stampState: "paper_stamp-notary"
+  - type: Sprite
+    sprite: DeltaV/Objects/Misc/stamps.rsi
+    state: stamp-notary
+  - type: StealTarget
+    stealGroup: RubberStampNotary
+    
+- type: entity
+  name: chief justice stamp
+  parent: RubberStampBase
+  id: RubberStampCJ
+  components:
+  - type: Stamp
+    stampedName: stamp-component-stamped-name-chiefjustice
+    stampedColor: "#6b2833"
+    stampState: "paper_stamp-notary"
+  - type: Sprite
+    sprite: DeltaV/Objects/Misc/stamps.rsi
+    state: stamp-cj # Change to CJ when sprited
```

appropriate use of CJ
```suggestion
    state: stamp-cj # Change to CJ when sprited

```